lib/rust/ensogl/component/list-view/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 //! ListView a displayed list of entries with possibility of selecting one and "choosing" by
 //! clicking or pressing enter - similar to the HTML `<select>`.
 
-#![recursion_limit = "512"]
+#![recursion_limit = "1024"]
```

Comment:
which macro requires it?

Reply:
Huh; while I was developing this PR, the compilation started failing at some point due to the recursion limit being exceeded, and Adam suggested that I increase the limit; but at the current state of the PR, this increase appears not needed anymore. I reverted it back to the original value.